### DISCUSSION TITLE
I would like to offer help in translating the docs to Japanese

### DISCUSSION THREAD

**Discussion by Naoto-Ida** [biomejs/biome#939] [General]
Hey y'all, I just read through the[ blog update on v1.4.0](https://biomejs.dev/blog/biome-wins-prettier-challenge). Congrats :)

I also noticed this:
> Translate the website in Japanese.

As a huge advocate of Biome at our organization, I'd love to help out in translating the docs!
Please feel free to reach out on any plans y'all have.

**Comment by ematipico** [comment]
Hi @Naoto-Ida , thank you for your help! You can follow this issue https://github.com/biomejs/biome/issues/880#issuecomment-1826477936

And tell which page you would like to translate.

We use that issue to coordinate the work.
